src/components/MainFeed/Post.jsx

```diff
@@ -16,7 +17,8 @@ export const Post = (props) => {
             </div>
             
             <img className='postPhoto' src={props.postPhoto} width="270px"/>
-            <div className='postComments'>{props.postComments}</div>
+            
+            <Comments></Comments>
```

Comment:
you  can use the short sytanx here for html because it doesnt have any children:
```suggestion
            <Comments/>
```